var/spack/repos/builtin/packages/python/package.py

```diff
@@ -1343,6 +1344,9 @@ def setup_dependent_run_environment(
         """Set PYTHONPATH to include the site-packages directory for the
         extension and any other python extensions it depends on.
         """
+        if self.spec.satisfies("platform=windows"):
+            env.prepend_path("PATH", dependent_spec.prefix.scripts)
```

Comment:
(question) PR description mentions "Scripts", this will do "scripts", which I assume will generally not matter since paths are not case sensitive on windows.

If the directory is expected to be called "Scripts", I have a slight inclination to refer to it that way.

Reply:
> which I assume will generally not matter since paths are not case sensitive on windows.

I was indeed relying on this behavior, will refactor to refer to the correctly named directory.